lvgl.pc.in

```diff
@@ -5,6 +5,6 @@ libdir=${prefix}/lib
 Name: lvgl
 Description: Light and Versatile Graphics Library
 URL: https://lvgl.io/
-Version: 9.1.0
+Version: @LVGL_VERSION@-@LVGL_VERSION_INFO@
```

Comment:
This is going to leave an extra `-` at the end of the version upon release (when `LVGL_VERSION_INFO` is empty)

Reply:
Great catch!

Reply:
@marex Can we do anything with that?

Reply:
I guess we can unconditionally drop the `-@LVGL_VERSION_INFO@` part ?

Reply:
I pushed such a change now , see https://github.com/lvgl/lvgl/compare/630ea60294159d4100101eee9e18e2dbd77597a1..f4c414a0d39ecdc22c54b9a53d8e3f1d93fc076a .

Reply:
Good for me, thank you!